exercises/TurnOnLED/TurnOnLED.ino

```diff
@@ -3,10 +3,17 @@
 const int LED=9;  // Define LED for pin 9
 
 void setup() {
-  pinMode(LED, OUTPUT)  // Set the LED pin as an output
+  pinMode(LED, OUTPUT); // Set the LED pin as an output
   digitalWrite(LED, HIGH);  // Set the LED pin high
 }
 
 void loop() {
   // Nothing to loop over
+  for (int i=100; i<=1000; i=i+100)
+{
+  digitalWrite(LED, HIGH); 
+  delay(i); 
+  digitalWrite(LED, LOW); 
+  delay(i);
+}
```

Comment:
Looks like you added code from the next exercise. I'll add a suggestion to take it out.
```suggestion
```